AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/managers/PluginHookManager.java

```diff
@@ -45,34 +59,27 @@ public ImportExportPlugin getImportPlugin(String name) {
         return importPlugins.get(name);
     }
 
-    private void loadImportPlugin(String name, Class<? extends ImportExportPlugin> clazz) {
+    private <T> void loadPlugin(HashMap<String, T> map, String name, Class<? extends T> clazz) {
         try {
-            importPlugins.put(name, clazz.newInstance());
-        } catch (InstantiationException | IllegalAccessException e) {
+            map.put(name, clazz.getConstructor().newInstance());
+        } catch (InstantiationException | NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
             e.printStackTrace();
-        } catch (NoClassDefFoundError ignored) {
```

Comment:
As it turns out, removing the check for this exception does break the plugin, e.g. if Essentials is not installed:

```java.lang.NoClassDefFoundError: com/earth2me/essentials/commands/WarpNotFoundException```

Ironically, this is the only plugin that throws such a fit.